connectors/aws/aws-sqs/src/main/java/io/camunda/connector/inbound/MessageMapper.java

```diff
@@ -45,7 +45,7 @@ public static SqsInboundMessage toSqsInboundMessage(final Message message) {
 
   private static Object toObjectIfPossible(final String body) {
     try {
-      return ObjectMapperSupplier.getMapperInstance().readValue(body, Object.class);
+      return ConnectorsObjectMapperSupplier.getCopy().readValue(body, Object.class);
```

Comment:
We can move it to a variable to avoid creating a copy for each static method call

Reply:
Actually, good point